src/azul/queues.py

```diff
@@ -285,7 +285,11 @@ def limit_timeout(timeout):
 
             # Do we have time left?
             remaining_time = start_time + timeout - time.time()
-            if remaining_time <= 0:
+
+            # One min allotted for eventual consistency on SQS
```

Comment:
Combine these three comments into one sequence of consecutive comment lines, aka a "block comment".